code/api/src/config/database.json

```diff
@@ -7,6 +7,14 @@
     "dialect": "postgres",
     "seederStorage": "sequelize"
   },
+  "test": {
+    "username": "postgres",
+    "password": null,
+    "database": "crate_test",
```

Comment:
So interesting to think about the test database just being a separate clone of the development database.

Reply:
Agreed!